### DISCUSSION TITLE
Suggestion for enabling Translation Memory Fill-up on Transifex

### DISCUSSION THREAD

**Discussion by adem4ik** [LMMS/lmms#7197] [General]
Please enable **Translation Memory Fill-up** setting for [Transifex project](https://app.transifex.com/lmms/lmms/dashboard/). It is explained [here](https://help.transifex.com/en/articles/6224817-setting-up-translation-memory-fill-up). That will save translators from needing to retranslate 100% matching strings.

**Comment by he29-net** [comment]
The page you linked mentions the feature is only available at the Growth subscription plan or higher. I'm pretty sure LMMS has a free account for open source projects, so we may not have this feature available.
